TensorFlow2/Segmentation/UNet3P/README.md

```diff
@@ -0,0 +1,318 @@
+
+Note: Training time includes time to load cuDNN in first iteration and the first epoch which take little longer as
+compared to later epochs because in first epoch tensorflow optimizes the training graph. In terms of latency and
+throughput it does not matter much because we have trained networks for 100 epochs which normalizes this during
+averaging.
+
+#### Inference performance results
+
+###### Inference performance: NVIDIA DGX A100 (1xA100 80G)
+
+| Batch Size |   XLA   |   AMP   | Latency Avg [ms] &darr; | Throughput Avg [img/s] &uarr; |
+|:----------:|:-------:|:-------:|:-----------------------:|:-----------------------------:|
+|     1      | &cross; | &cross; |          59.54          |             16.79             |
+|     1      | &check; | &cross; |          70.59          |             14.17             |
+|     1      | &cross; | &check; |          56.17          |             17.80             |
+|     1      | &check; | &check; |          55.54          |             18.16             |
+|     16     | &cross; | &cross; |         225.59          |             70.93             |
+|     16     | &check; | &cross; |         379.93          |             43.15             |
+|     16     | &cross; | &check; |         184.98          |             87.02             |
+|     16     | &check; | &check; |         153.65          |             103.6             |
+
+Inference results are tested on single gpu. Here data generator type does not matter because only prediction time
+is calculated and averaged between 5 runs.
+
+## Inference Demo
+
+Model output can be visualized from Jupyter Notebook. Use below command to start Jupyter Lab on port `8888`.
+
+```
+jupyter lab --no-browser --allow-root --ip=0.0.0.0 --port=8888
+```
+
+While starting container we mapped system port `5012` to `8888` inside docker.
+> Note: Make sure you have server network ip and port access in case you are working with remote sever.
+
+Now in browser go to link `http://<server ip here>:5012/` to access Jupyter Lab.
+Open [predict.ipynb](predict.ipynb) notebook and rerun the whole notebook to visualize model output.
+
+There are two options for visualization, you can
+
+1. Visualize from directory
+
+It's going to make prediction and show all images from given directory. Useful for detailed evaluation.
+
+2. Visualize from list
+
+It's going to make prediction on elements of given list. Use for testing on specific cases.
+
+For custom data visualization set `SHOW_CENTER_CHANNEL_IMAGE=False`. This should set True for only UNet3+ LiTS data.
+
+For further details on visualization options see [predict.ipynb](predict.ipynb) notebook.
+
+## Known issues
+
+There are no known issues in this release.
+
+## Release notes
+
+### Changelog
+
+Feb 2023
+
+- Initial release
+
+We appreciate any feedback so reporting problems, and asking questions are welcomed here.
+
+Licensed under [MIT License](LICENSE)
```

Comment:
can we change to apache?

Reply:
License updated to Apache and its reference is also updated accordingly.